event-based-simulators/main/cumulocityAPI.py

```diff
@@ -7,19 +7,28 @@
 
 MOCK_REQUESTS = os.environ.get('MOCK_C8Y_REQUESTS') or 'false'
 
-user_and_pass_bytes = base64.b64encode((C8Y_TENANT + "/" + C8Y_USER + ':' + C8Y_PASSWORD).encode('ascii')) # bytes
-user_and_pass = user_and_pass_bytes.decode('ascii') # decode to str 
+user_and_pass_bytes = base64.b64encode((C8Y_TENANT + "/" + C8Y_USER + ':' + C8Y_PASSWORD).encode('ascii'))  # bytes
+user_and_pass = user_and_pass_bytes.decode('ascii')  # decode to str
 
 C8Y_HEADERS = {
     'Content-Type': 'application/json',
     'Accept': 'application/json',
     'Authorization': 'Basic ' + user_and_pass
 }
-
+MEASUREMENTS_COLLECTION_HEADERS = {
+    'Content-Type': 'application/vnd.com.nsn.cumulocity.measurementcollection+json',
+    'Accept': 'application/json',
+    'Authorization': 'Basic ' + user_and_pass
+}
```

Comment:
is MEASUREMENTS_COLLECTION_HEADERS used anywhere?

Reply:
If it is not needed, it should be removed. We can add it, once it is required. Or do you think you will need it within this PR?